src/connector/src/connector_common/connection.rs

```diff
@@ -98,6 +98,30 @@ impl Connection for KafkaConnection {
     }
 }
 
+use rdkafka::config::RDKafkaLogLevel;
+
+pub fn read_kafka_log_level() -> RDKafkaLogLevel {
+    let log_level =
+        std::env::var("RISINGWAVE_KAFKA_LOG_LEVEL").unwrap_or_else(|_| "INFO".to_owned());
```

Comment:
If we need to add this, prefer returning an `Option` instead of INFO here. As mentioned below, it can also be configured via `RUST_LOG`